sheets/umask

```diff
@@ -5,8 +5,8 @@
 # mode creation mask") for only the current user, and only his or her current
 # session. The (one) leading zero is optional, unless you otherwise need it.
 #
-# This umask setting is actually recommended for security by RHEL, and is also
-# mentioned and I believe recommended in the Arch Linux Wiki.
+# This umask setting is actually recommended for security by major Linux distributions
```

Comment:
I've only seen this recommended by RHEL, and so Arch. To my knowledge, the default, with no apparent official recommendation to the contrary, is to default to 644 and 755 for files and directories, respectively. I'm aware of this because I've always changed my _umask_ to `0077`.